### PR TITLE
KEYCLOAK-19145 remove POSTGRES_HOST and add text to specify the proxy…

### PR DESCRIPTION
**Pull request by jonathanvila**
https://issues.redhat.com/browse/KEYCLOAK-19145

Remove the mention of POSTGRES_HOST  and detail the usage of a proxy to connect to the external database.

### CODE REVIEW

File: server_installation/topics/operator/external-database.adoc [hunk 1]
```diff
@@ -35,11 +33,12 @@ endif::[]
 The other properties work in the same way for a hosted or external database. Set them as follows:
 
 * `POSTGRES_DATABASE` - Database name to be used.
-* `POSTGRES_HOST` - The name of the `Service` used to communicate with a database. Typically `keycloak-postgresql`.
 * `POSTGRES_USERNAME` - Database username
 * `POSTGRES_PASSWORD` - Database password
 * `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
 
+The Keycloak Operator will create a Service to connect to the external database, tipycally `keycloak-postgresql`, that will connect to Keycloak to the Database, containing the IP or Service name specified by the POSTGRES_EXTERNAL_ADDRESS.
```
Comment (vmuzikar) [inline server_installation/topics/operator/external-database.adoc]: @jonathanvila I'd rephrase this a bit to make it even more explicit. :)
```suggestion
The {project_operator} will create a Service to connect to the external database, typically `keycloak-postgresql`. This Service points to the actual external database (as specified by `POSTGRES_EXTERNAL_ADDRESS`) and {project_name} then connects to the database through this operator managed service.
```
WDYT?

Comment (jonathanvila) [reply server_installation/topics/operator/external-database.adoc]: :+1:

**Review by vmuzikar — DISMISSED** [review]
@jonathanvila LGTM, thank you!

@andymunro Could you please review/merge when you have a chance?

File: server_installation/topics/operator/external-database.adoc (hunk 1 above)
Comment (andymunro) [inline server_installation/topics/operator/external-database.adoc]: The Operator will create a Service that connects to an external database, such as `keycloak-postgresql`, that connects {project_name} to the Database that contains the IP address or Service name that is specified by the POSTGRES_EXTERNAL_ADDRESS.

**Review by andymunro — CHANGES_REQUESTED** [review]
Just one change. Thanks, @jonathanvila

File: server_installation/topics/operator/external-database.adoc (hunk 1 above)
Comment (jonathanvila) [reply server_installation/topics/operator/external-database.adoc]: :+1:

**Review by andymunro — DISMISSED** [review]

File: server_installation/topics/operator/external-database.adoc (hunk 1 above)
Comment (vmuzikar) [reply server_installation/topics/operator/external-database.adoc]: @andymunro @jonathanvila The sentence IMHO now have a completely different meaning. E.g. it implies that `keycloak-postgresql` is the name of the external database, not the Service. And the second part of the sentence IMHO doesn't now make much sense, what connects to what. Maybe we could do something like this:

```
The Operator will create a Service named `keycloak-postgresql`. This Service is configured by the Operator to expose the external database based on the content of `POSTGRES_EXTERNAL_ADDRESS`. {project_name} uses this Service to connect to the Database, that means it does not connect to the Database directly but rather through this Service.
```

**Review by vmuzikar — COMMENTED** [review]
@andymunro @jonathanvila Please see https://github.com/keycloak/keycloak-documentation/pull/1274#discussion_r744881275.

Sorry for the late comment. I just noticed the sentence was rewritten.

File: server_installation/topics/operator/external-database.adoc (hunk 1 above)
Comment (andymunro) [reply server_installation/topics/operator/external-database.adoc]: Looks good except "that means" should be "which means"

Comment (jonathanvila) [reply server_installation/topics/operator/external-database.adoc]: :+1:

**Review by vmuzikar — APPROVED** [review]
